Codeblock has a case sensitive issue

In your https://johnnyreilly.com/docusaurus-using-fontaine-to-reduce-custom-font-cumulative-layout-shift article there's a case sensitive issue with  this codeblock section.

```
\\ ...
 plugins: [
              fontaine.fontaineTransform.webpack({
                fallbacks: [
\\ ...
```

Should be ` fontaine.FontaineTransform.webpack`

Had me scratching my head when I got a docusaurus build error, but caught this. :) 


```
[ERROR] TypeError: Cannot read properties of undefined (reading 'webpack')
    at Object.configureWebpack (/workspace/didocs/docusaurus.config.js:187:42)
    at applyConfigureWebpack (/workspace/didocs/node_modules/@docusaurus/core/lib/webpack/utils.js:133:43)
    at /workspace/didocs/node_modules/@docusaurus/core/lib/commands/build.js:155:62
    at Array.forEach (<anonymous>)
    at buildLocale (/workspace/didocs/node_modules/@docusaurus/core/lib/commands/build.js:149:13)
    at async tryToBuildLocale (/workspace/didocs/node_modules/@docusaurus/core/lib/commands/build.js:41:20)
    at async mapAsyncSequential (/workspace/didocs/node_modules/@docusaurus/utils/lib/jsUtils.js:34:24)
    at async Command.build (/workspace/didocs/node_modules/@docusaurus/core/lib/commands/build.js:76:21)
```


P.S. Awesome blog I use your swc-loader, preload fonts, and fontaine tutorials. Gonna give the prefetch priority one a try too.

--- a/blog-website/blog/2022-12-01-docusaurus-using-fontaine-to-reduce-custom-font-cumulative-layout-shift/index.md
+++ b/blog-website/blog/2022-12-01-docusaurus-using-fontaine-to-reduce-custom-font-cumulative-layout-shift/index.md
@@ -79,7 +79,7 @@ const config = {
         configureWebpack(_config, _isServer) {
           return {
             plugins: [
-              fontaine.fontaineTransform.webpack({
+              fontaine.FontaineTransform.webpack({
                 fallbacks: [
                   'system-ui',
                   '-apple-system',

